services/github/schemas/v1/taskcluster-render-yaml-input.yml

```diff
@@ -0,0 +1,76 @@
+$schema: "/schemas/common/metaschema.json#"
+title: "Render .taskcluster.yml input"
+description: |
+  Render .taskcluster.yml for given event.
+type:         object
+properties:
+  body:
+    title: Taskcluster.yml contents
+    description: The contents of the .taskcluster.yml
+    type: string
+  organization:
+    type: string
+    description: The organization that owns the repository.
+    pattern: "^[a-zA-Z0-9_-]+$"
+    default: TaskclusterRobot
```

Comment:
why is there a default here? is this what the github service users as a default if no org is specified? when does GitHub not have an org in an event message?

Reply:
good question. This endpoint was intended to be used for debug purposes, so you could send your .taskcuster.yml as string and one of the events to be faked, that would produce task definitions using mocked data. 
We can remove default value from schema, but we'll still use something similar in code when fake event would be mocked